README.md

```diff
@@ -78,7 +78,14 @@ StellarGraph is built on [TensorFlow 2](https://tensorflow.org/) and its [Keras
 
 [The numerous detailed and narrated examples](https://github.com/stellargraph/stellargraph/tree/master/demos/) are a good way to get started with StellarGraph. There is likely to be one that is similar to your data or your problem (if not, [let us know](#getting-help)).
 
-You can start working with the examples immediately in Google Colab or Binder by clicking the ![](https://colab.research.google.com/assets/colab-badge.svg) and ![](https://mybinder.org/badge_logo.svg) badges within each Jupyter notebook. You can also run them locally, after installing StellarGraph. One of the following might be appropriate:
+You can start working with the examples immediately in Google Colab or Binder by clicking the ![](https://colab.research.google.com/assets/colab-badge.svg) and ![](https://mybinder.org/badge_logo.svg) badges within each Jupyter notebook.
+
+Alternatively, you can run download a local copy of the demos and run them using `jupyter`. The demos can be downloaded by cloning the `master` branch of this repo, or by using the [`svn` command-line tool](https://subversion.apache.org/) to download the specific directory:
+```bash
+svn export https://github.com/stellargraph/stellargraph/branches/master/demos
```

Comment:
wow, it feels like about 10 years since I've used subversion - amazing that this is the easiest way!